zrml/court/src/juror.rs

```diff
@@ -6,6 +6,7 @@ use crate::JurorStatus;
     Clone,
     Debug,
     PartialEq,
+    parity_scale_codec::MaxEncodedLen,
```

Comment:
[nitpick] MaxEncodedLen imports appear a bit inconsistent in this PR (sometimes imported directly, sometimes not). Is there some pattern that I don't see?

Reply:
The idea was to use the path whenever it is used only once and otherwise use name bindings. But I managed to get it inconsistent in that regard anyways :sweat_smile: